storybook/storybook-react/src/components/Link/Link.docs.mdx

```diff
@@ -18,10 +18,6 @@ Op zichzelf staande koppelingen staan direct na inhoud. Ze mogen niet in zinnen
 
 <Canvas of={LinkStories.Standalone} />
 
-### Standalone Bold
-
-<Canvas of={LinkStories.StandaloneBold} />
```

Comment:
Perhaps Figma should be updated?

Reply:
Yeah, the documentation on link variants are a little confusing at the moment. I've added a comment in the Figma file.